test/unit/test.step.js

```diff
@@ -95,111 +106,108 @@ describe('Step', () => {
   });
 
   describe('bindAdvance()', () => {
```

Comment:
These changes are mainly due to the fact that `Step.isOpen` is no longer reading `this.el.hidden` to determine that.

... but I also took the opportunity to abstract some of the duplication to a `beforeEach` 😄.